mem-18-v.xml

```diff
@@ -5927,7 +5927,7 @@ Katso idioomaattinen ilmaisu, jota käytetään käskemään jotakuta liikkumaan
       <column name="entry_name">vIt</column>
       <column name="part_of_speech">v:t,klcp1</column>
       <column name="definition">tell the truth</column>
-      <column name="definition_de">wahrheit sprechen</column>
+      <column name="definition_de">die Wahrheit sprechen</column>
```

Comment:
Fixed in https://github.com/De7vID/klingon-assistant-data/commit/2a1ec773da8be530e5e5c32fde608367f177f682.